### PR TITLE
comment activities and adapter

### LINKED ISSUE #180: KDoc for activities and adapter packages

### PR DESCRIPTION
**Pull request by lpatel99**

### CODE REVIEW

File: app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt [hunk 1]
```diff
@@ -47,6 +49,9 @@ class ProjectAdapter(
         }
     }
 
+    /**
+     * Make dataset listen for changes to the projects.
```
Comment (arnauds5) [inline app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt]: It also fetch all projects from the database.

**Review by arnauds5 — APPROVED** [review]

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (PaulCoral) [inline .idea/misc.xml:3]: not needed

File: .idea/runConfigurations.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RunConfigurationProducerService">
+    <option name="ignoredProducers">
+      <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
+      </set>
+    </option>
+  </component>
+</project>
```
Comment (PaulCoral) [inline .idea/runConfigurations.xml:10]: not needed

**Review by PaulCoral — APPROVED** [review]
Everything seems OK, mostly reformatting, optimizing imports, and removing warnings.
I added a few notes about certainly **unnecessary files**

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```
Comment (PaulCoral) [inline gradle/wrapper/gradle-wrapper.properties:6]: is this needed ?
leave it there if you do not know

**Pull request merged**